pkg/plugin/v3/scaffolds/init.go

```diff
@@ -126,7 +126,6 @@ func (s *initScaffolder) scaffold() error {
 		&webhook.Kustomization{},
 		&webhook.KustomizeConfigWebhook{},
 		&webhook.Service{},
```

Comment:
IMO, I think we need also move these ones to the webhook plugin as well since they are specific to the webhook plugin and are not useful to projects which as not in the webhooks. I mean, it shows to be the same scenario.

Reply:
Yes, I also thinks the same..  Should I move all the webhook related scaffolding in this PR itself? Or should we wait for discussion to be finalized on #1644 ?

Reply:
I understood that we agreed to move forward with into the bug triage. However, let's see if @estroz would like to hold on that until we finalized on #1644.